autogen/safer-cluster/main.tf.tmpl

```diff
@@ -185,6 +185,9 @@ module "gke" {
   // We enable Workload Identity by default.
   identity_namespace = "${var.project_id}.svc.id.goog"
 
+  // Enabling mesh certificates requires Workload Identity
+  enable_mesh_certificates = var.enable_mesh_certificates
```

Comment:
Please wrap this in {% if autopilot_cluster != true %} so it doesn't render for autopilot clusters.